runner/docker/Dockerfile.models

```diff
@@ -0,0 +1,3 @@
+FROM alpine:3
+
+COPY ./models/ /models/
```

Comment:
Ideally I'd write sth like:

```
FROM livepeer/ai-worker:live-app-comfyui AS builder
RUN runner/dl_checkpoints.sh --tensorrt

FROM alpine:3
COPY --from:builder /models/ /models/
```

Reply:
it could just be
```dockerfile
FROM scratch
COPY ...
```
no need to even go for alpine until we start building the models inside docker itself.